src/yang/ietf-bgp-policy.yang

```diff
@@ -248,6 +248,32 @@ module ietf-bgp-policy {
           }
         }
       }
+      container next-hop-sets {
+        description
+          "Definition of a list of IPv4 or IPv6 next-hops which can
+           be matched in a routing policy.";
+
+        list next-hop-set {
+          key "name";
+          description
+            "List of defined next-hop sets for use in policies.";
+
+          leaf name {
+            type string {
+              length "1..79";
```

Comment:
Why is this being constrained?

Reply:
Good question. OC has this in their model.